doc/How-Tos/Hosting/Hosting-setup.md

```diff
@@ -0,0 +1,33 @@
+# How to get started with hosting
+
+Hosting can be used to register services that will be accessible throughout the application via dependency injection. This tutorial will walk you through the critical pieces needed to leverage hosting in your application. 
+
+NOTE: If you used the `dotnet new` template to create the solution, some of the steps outlined here are unecessary. You can see whether your application already creates the IHost interface using the CreateDefaultBuilder method by looking for a generated `App.xaml.host.cs` file.
+
+## Step-by-steps
+1. Install the Uno.Extensions.Hosting library from NuGet, making sure you reference the package that is appropriate for the flavor of Uno in use.
```

Comment:
Be specific here as to which packages to install - don't forget some readers will be completely new and won't understand what you mean, so will try to install both packages